packages/mdc-textfield/outline/README.md

```diff
@@ -0,0 +1,40 @@
+<!--docs:
+title: "Text Field Outline"
+layout: detail
+section: components
+excerpt: "The outline is a border around the text field"
+iconId: text_field
+path: /catalog/input-controls/text-fields/outline/
+-->
+
+# Text Field Outline
+
+The outline is a border around all sides of the text field. This is used for the Outlined variation of Text Fields.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/text-fields.html#text-fields-field-variations">Material Design guidelines: Text Field Variations</a>
+  </li>
+</ul>
+
+## Usage
+
+#### MDCTextFieldOutline API
+
+##### MDCTextFieldOutline.foundation
+
+MDCTextFieldOutlineFoundation. This allows the parent MDCTextField component to access the public methods on the MDCTextFieldOutlineFoundation class.
+
+### Using the foundation class
+
+Method Signature | Description
+--- | ---
+setOutlinePathAttr(value: string) => void | Sets the SVG path of the outline element
```

Comment:
Should this specifically say it sets the `d` attribute of the path within the outline SVG?